documentation/modules/con-securing-client-authentication.adoc

```diff
@@ -67,7 +79,30 @@ data:
   user.password: # Password for protecting the PKCS #12 archive file
 ----
 
-== SCRAM-SHA-512 Authentication
+== TLS client authentication using an external certificate
+
+To use TLS client authentication using a certificate issued outside the User Operator, you set the `type` field to `tls-external`.
+A secret and credentials are not created for the user.
+
+.An example `KafkaUser` with TLS client authentication enabled
+[source,yaml,subs="attributes+"]
+----
+apiVersion: {KafkaUserApiVersion}
+kind: KafkaUser
+metadata:
+  name: my-user
+  labels:
+    strimzi.io/cluster: my-cluster
+spec:
+  authentication:
+    type: tls-external
+  # ...
+----
+
+
+.Additional resources
+
```

Comment:
Missing additional resources.

Reply:
I moved them, but missed the title. Removed.